src/odemis/driver/test/actuator_test.py

```diff
@@ -1653,10 +1668,22 @@ def move_to_initial_position():
         time.sleep(0.1)
         cancelled = f.cancel()
         self.assertTrue(cancelled)
-        # Asset that initial position is not reached (during focus adjustment)
+        # Assert that initial position is not reached (during focus adjustment)
         logging.debug(focus.position.value)
         test.assert_pos_not_almost_equal(focus.position.value, initial_foc_pos, atol=ATOL_LENS)
 
+        move_to_initial_position()
+        target_pos = {'z': self.target_value}  # To move stage first
+        initial_stage_pos = stage.position.value.copy()
+        f = stage.moveAbs(target_pos)
+        time.sleep(0.1)
+        cancelled = f.cancel()
+        self.assertTrue(cancelled)
+        # Assert that stage position is neither initial nor target positions
+        logging.debug(stage.position.value)
+        test.assert_pos_not_almost_equal(initial_stage_pos, stage.position.value, atol=ATOL_STAGE)
+        test.assert_pos_not_almost_equal(stage.position.value, target_pos, match_all=False, atol=ATOL_STAGE)
+
```

Comment:
Alas this has the same behaviour as the 3dof simulator were. It moves instantly without simulating a gradual move. Haven't you applied this fix on the 5dof simulator @pieleric?